### ISSUE TITLE
bug(Kubernetes): duplicate Node

### ISSUE THREAD

**Issue by axel7083** [containers/podman-desktop#9784]
### Bug description

I often see twice the node resource for my kind cluster as show bellow

![Image](https://github.com/user-attachments/assets/8a140370-34b7-417c-9bcb-f4a775e3f193)

### Operating system

Windows 11

### Installation Method

None

### Version

next (development version)

### Steps to reproduce

I am not able to have a full reproducer, as this seems a bit random but the steps are usually as follow

1. assert kind container exited
2. start podman desktop
3. wait a bit ?
4. start the kind container
5. wait a bit
6. go to Kubernetes node page
7. assert sometime twice the same node

### Relevant log output

N/A

**Comment by feloy** [comment]
When adding an onject, removing the previous object with the same UID is done for pods (https://github.com/containers/podman-desktop/blob/main/packages/main/src/plugin/kubernetes/contexts-manager.ts#L314), it would be helpful to do the same for other resources as well.

I would expect that, in normal conditions, the same object is not added twice to the object. I suspect that some race condition happen between the state (where the objects sent by the informers are stored) and the informers (which can be stopped/restarted and thus can resend ADDED objects).

**Comment by axel7083** [comment]
cc @feloy can we close this issue?

**Comment by feloy** [comment]
yes, it should be fixed in experimental mode, as the cache is handled by the kubernetes-client library itself
